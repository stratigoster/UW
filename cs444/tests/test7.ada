PACKAGE foo IS
  s : STRING := "Bye World""" & "Hi Heaven" & 10 & "Lucky 7";
--                                            ^^
-- type mismatch: trying to append an integer to a string
END foo;
