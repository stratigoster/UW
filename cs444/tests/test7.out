***ERROR***:2:45:	Can only concatenate strings

------
Compilation error
------
